code-challenges.test.js

```diff
@@ -37,6 +73,32 @@ const arrayOfWords2 = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach"]
 const letterE = "e"
 // Expected output: ["Cherry", "Blueberry", "Peach"]
 
+const found = (array) => {
+    for (let i = 0; i < array.length; i++){
+    array.find(element => 'a')
+    }
+    return array
+}
+
+describe("found", () => {
+
+    const arrayOfWords1 = ["Apple", "Banana", "Plum", "Orange", "Kiwi"]
+    const letterA = "a"
+    // Expected output: ["Apple", "Banana", "Orange"]
+    const arrayOfWords2 = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach"]
+    const letterE = "e"
+
+ 
+    it("returns a string that says hiCreate a function that takes in a string and returns a coded message. The coded message converts 'a' to 4, 'e' to 3, 'i' to 1, and 'o' to 0.", () => {
+  
+      
+      expect(found(arrayOfWords1)).toEqual(["Apple", "Banana", "Orange"])
+     
```

Comment:
Your it statement seems to have been copied from another test and not refactored. It is great to reuse code and "work smarter not harder" but often copying code requires extra attention to detail.